documentation/docs/strategies/avellaneda-market-making.md

```diff
@@ -58,43 +60,99 @@ This strategy implements the market making strategy described in the classic pap
 !!! note "Approximation only"
     The description below is a general approximation of this strategy. Please inspect the strategy code in **Trading Logic** above to understand exactly how it works.
 
-Coming soon.
+### Description
 
- <!-- 
+The strategy continuously calculates optimal positioning of a market maker's buy and sell limit orders within an order book, based on the following information:
 
-### `parameters_based_on_spread`
+- **Current order book liquidity**
+- **Asset price volatility**
+- **Desired portfolio allocation (target inventory)**
+- **Trading session timeframe**
+- **Risk factor (user choice)**
 
-The parameter acts as a toggle between beginner (parameters_based_on_spread=True) and expert mode (False). When equal to true, the strategy will require min spread, max spread, volatility multiplier, and inventory risk aversion, while if set to False, it will only ask for risk_factor, order_book_depth_factor, and order_amount_shape_factor..
+There is two main values that are calculated by the model, based on the factors mentioned above:
 
-### **Advanced parameters**
+- **Reserved price**: A price different from the market mid price, that will be used as reference to create orders.
+- **Optimal spread**: The best possible spread from the *reserved price* where the orders will be created.
 
-All 3 parameters `order_book_depth_factor`, `risk_factor` and `order_amount_shape_factor` are customizable only in expert mode (toggled by `parameters_based_on_spread`=False).
+#### Reserved Price
```

Comment:
Use the term `Reservation Price` throughout the docs instead of `Reserved Price` globally since that's what is used in the original paper.